apps/openmw/mwrender/sky.cpp

```diff
@@ -1472,7 +1482,7 @@ void SkyManager::createRain()
 
     mRainNode = new osg::Group;
 
-    mRainParticleSystem = new osgParticle::ParticleSystem;
+    mRainParticleSystem = new NifOsg::ParticleSystem;
```

Comment:
Why isn't mRainParticleSystem type changed to NifOsg one anyway?

Reply:
just in order to be consistant with other partsys